web/yo/app/scripts/controllers/gene.js

```diff
@@ -321,18 +326,15 @@ angular.module('oncokbApp')
                 $scope.status.hasReviewContent = false;
                 $scope.status.mutationChanged = false;
                 var userNames = [];
-                if(mainUtils.needReview($scope.gene.summary_uuid)) {
-                    $scope.status.hasReviewContent = true;
-                    userNames.push($scope.gene.summary_review.get('updatedBy'));
-                }
-                if(mainUtils.needReview($scope.gene.type_uuid)) {
-                    $scope.status.hasReviewContent = true;
-                    userNames.push($scope.gene.type_review.get('updatedBy'));
-                }
-                if(mainUtils.needReview($scope.gene.background_uuid)) {
-                    $scope.status.hasReviewContent = true;
-                    userNames.push($scope.gene.background_review.get('updatedBy'));
-                }
+                var geneEvisMapping = [{uuid: $scope.gene.summary_uuid, review: $scope.gene.summary_review},
```

Comment:
This works, you can also use `['gene', 'summary', 'background']` and later loop through them and use 
```
if(mainUtils.needReview($scope.gene[key + '_uuid'])) {                    
    $scope.status.hasReviewContent = true;
    userNames.push($scope.gene[key + '_review'].get('updatedBy'));   
}